From f60d4ba0f6145b960eff085b112a0fb5eb5aafe3 Mon Sep 17 00:00:00 2001
From: Matthias Klumpp <matthias@tenstral.net>
Date: Fri, 8 Jan 2016 20:24:28 +0100
Subject: [PATCH] apt: Ensure DEP-11 files stay gzip compressed

We late might want to switch to XZ compression on the server and lz4
compression on the client, but for now we tell APT that we explicitly
want gz compression, so newer releases of APT don't try to recompress
our files to formats we don't support.
---
 contrib/apt-conf/50appstream | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contrib/apt-conf/50appstream b/contrib/apt-conf/50appstream
index 913490b..b483143 100644
--- a/contrib/apt-conf/50appstream
+++ b/contrib/apt-conf/50appstream
@@ -11,6 +11,7 @@ Acquire::IndexTargets {
         flatMetaKey "Components-$(NATIVE_ARCHITECTURE)";
         flatDescription "$(RELEASE) DEP-11 Metadata";
         KeepCompressed "true";
+        KeepCompressedAs "gz";
     };
 
     # Normal-sized icons for GUI components described in the DEP-11
@@ -23,6 +24,7 @@ Acquire::IndexTargets {
         flatMetaKey "icons-64x64";
         flatDescription "$(RELEASE) DEP-11 64x64 Icons";
         KeepCompressed "true";
+        KeepCompressedAs "gz";
     };
 
     # Icons suitable for HiDPI displays, for GUI components described in
@@ -35,6 +37,7 @@ Acquire::IndexTargets {
         flatMetaKey "icons-128x128";
         flatDescription "$(RELEASE) DEP-11 128x128 Icons";
         KeepCompressed "true";
+        KeepCompressedAs "gz";
         DefaultEnabled "false";
     };
 };
